src/Services/UserManagementService/UsersManagementService.DAL/Entites/PagedUsersRequest.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace UsersManagementService.DAL.Entites;
+
+public class PagedUsersRequest
```

Comment:
naming "Request" is more suitable for Presentation level. For DAL something like "Filter" is better